test/api/v3/integration/groups/POST-groups_groupId_leave.js

```diff
@@ -0,0 +1,86 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /groups/:groupId/leave', () => {
+  let user;
+  let userToInvite;
+  let group;
+  let groupName = 'Test Public Guild';
+  let groupType = 'guild';
+
+  beforeEach(async () => {
+    user = await generateUser({balance: 2});
+    userToInvite = await generateUser();
+    group = await user.post('/groups', {
+      name: groupName,
+      type: groupType,
+    });
+    await user.post(`/groups/${group._id}/invite`, {
+      uuids: [userToInvite._id],
+    });
+  });
+
+  describe('leaving guilds', () => {
```

Comment:
context is just an alias for describe, so functionally, they do the same thing.

In general, our style is to have a describe block describe a method (or in this case, a route), and context blocks describe a specific circumstance for differing behavior.

So yes, this should be a context block.